src/uwtools/config/tools.py

```diff
@@ -277,8 +277,10 @@ def _validate_format_supplemental(
         if isinstance(supplemental_cfg, Config)
         else _ensure_format(desc=pre, config=supplemental_cfg)
     )
-    if sc_fmt != config_fmt:
-        raise UWError("%s format %s must match input format %s" % (pre, sc_fmt, config_fmt))
+    if sc_fmt not in (FORMAT.yaml, config_fmt):
+        raise UWError(
+            "%s format %s must be input format %s or %s" % (pre, sc_fmt, config_fmt, FORMAT.yaml)
```

Comment:
```suggestion
            "%s format %s must be %s or input format %s" % (pre, sc_fmt, FORMAT.yaml, config_fmt)
```
Apologies -- The yaml vs input format order you had before was better because it made it clear that "input format" only applies to one of the two accepted formats. I think it *is* better that we reference `FORMAT.yaml` instead of hardcoding the string, but your original order was better.